image_multiclass_trainer.py

```diff
@@ -56,22 +63,22 @@ def log_metrices(self, split):
 
     def training_step(self, batch, batch_idx):
         loss = self.shared_step(batch, batch_idx, 'train')
-        self.log('train_loss', loss.item(), on_step=False, on_epoch=True, logger=True)
+        self.log('train_loss', loss.detach(), on_step=False, on_epoch=True, logger=True)
```

Comment:
If I understand correctly, you will log a tensor as part of the train_loss, why not to return a scalar?

Reply:
torch lightning guide said that using .item() is slowing model training.
see:
https://lightning.ai/docs/pytorch/stable/advanced/speed.html#things-to-avoid